src/Database/Esqueleto/PostgreSQL.hs

```diff
@@ -363,3 +371,75 @@ filterWhere aggExpr clauseExpr = ERaw noMeta $ \_ info ->
     in ( aggBuilder <> " FILTER (WHERE " <> clauseBuilder <> ")"
        , aggValues <> clauseValues
        )
+
+newtype PgValuesExprs a = PgValuesExprs { unPgValuesExprs :: NE.NonEmpty a }
+
+instance (ToSomeValues a, Ex.ToAliasReference a, Ex.ToAlias a) => Ex.ToFrom (PgValuesExprs a) a where
+  toFrom = fromValues . unPgValuesExprs
```

Comment:
Is there a reason why you are going through `ToFrom` instead of just just making `values === fromValues`?

`ToFrom` is mostly there to support `SqlSetOperation` as a restricted subset and backwards compatibility with the `*Join` datatypes

Reply:
Ah, no reasons, this is just a lack of my knowledge of code-base and how it works in day when I wrote that :)
Thanks a lot for note, this is more clear for me now!

I have [fixed](https://github.com/bitemyapp/esqueleto/pull/284/commits/52373e5c430f5008ad4789528f519d60c9c2b39f) that :)